slick-testkit/src/codegen/resources/dbs/mysql.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE `slick_test`.`schema_name`( id INT NOT NULL  );
```

Comment:
Instead of `slick_test` or `schema_name` ?
The default table used for tests in Slick is called `slick_test`, I explicitly appended that so the Schema gets added by the Model builder in the table definition. 

Changing `slick_test` to `catalog_name` would cause a codegen failure because the schema(catalog_name) doesn't exist.(Unless we create and drop another database in testkit-reference.conf ? -Seems like overkill) 

I could rename the table schema_name to catalog_name